utils/userFriendlyErrorMessages.ts

```diff
@@ -0,0 +1,12 @@
+export interface ErrorMessages {
```

Comment:
This is just an example of how we could map specific error codes to user-friendly messages.